glomap/estimators/global_rotation_averaging.cc

```diff
@@ -56,6 +56,9 @@ bool RotationEstimator::EstimateRotations(
       image.cam_from_world.rotation = Eigen::Quaterniond(AngleAxisToRotation(
           rotation_estimated_.segment(image_id_to_idx_[image_id], 3)));
     }
+    // Restore the prior position (t = -Rc = R * R_ori * t_ori = R * t_ori)
```

Comment:
R * R_ori * t_ori = R * t_ori
The equation only holds when R_ori is identity.
However, EstimateRotations will be called twice in GlobalMapper::Solve.